app/_src/kubernetes-ingress-controller/guides/requests/rewrite-host.md

```diff
@@ -105,14 +122,58 @@ You can set the Host header explicitly if needed by disabling `konghq.com/preser
 
 There are three options to rewrite the default path handling behavior:
```

Comment:
I guess this would also have to be changed given that we can have 4 options in 3.2+.

```suggestion
Users have the following options to rewrite the default path handling behavior:
```